Source/Fields/LMCField.hh

```diff
@@ -72,6 +72,7 @@ namespace locust
             virtual std::vector<double> GetDopplerFrequency(int l, int m, int n, std::vector<double> tKassParticleXP) {return {0.};};
             virtual std::vector<double> GetNormalizedModeField(int l, int m, int n, std::vector<double> tKassParticleXP) {return {0.};};
             virtual std::vector<double> GetNormalizedModeField(int l, int m, int n, std::vector<double> tKassParticleXP, bool includeOtherPols) {return {0.};};
+	    virtual std::vector<double> GetNormalizedModeField(int l, int m, int n, std::vector<double> tKassParticleXP, bool includeOtherPolsi, bool teMode) {return {0.};};
```

Comment:
Similarly, can these two GetNormalizedModeField() functions be combined into just one function?